File: TexasPoker/ExportProtocol/ExportProtocol/bin/Debug/javacode/requesters/CreateRoleRqst.java

//============================================
//--4>:
//    Exported by ExportProtocol
//
//    此代码为工具根据配置自动生成, 建议不要修改
//
//============================================

package requesters;
import java.nio.ByteBuffer
import comm.BaseRqst;
import utils.JkTools;

public class CreateRoleRqst extends BaseRqst {
	private String name;
	private byte sex;
	private byte job;
	public void fromBytes(ByteBuffer bytes) {
		super.fromBytes(bytes);
		name = JkTools.readString(bytes);
		sex = bytes.get();
		job = bytes.get();
	}
}

File: TexasPoker/ExportProtocol/ExportProtocol/bin/Debug/cscode/requesters/CreateRoleRqst.cs

//============================================
//--4>:
//    Exported by ExportProtocol
//
//    此代码为工具根据配置自动生成, 建议不要修改
//
//============================================

namespace Requesters
{
	using System;
	using UnityEngine;
	using Socket;
	using Socket.Net;

	public class CreateRoleRqst : BaseRqst
	{
	public CreateRoleRqst(
			string name,
			byte sex,
			byte job
			)
			: base(10002)
		{
			m_Bytes.WriteString(name);
			m_Bytes.WriteByte(sex);
			m_Bytes.WriteByte(job);
		}
	}
}
